configs/sites/orion/packages.yaml

```diff
@@ -73,12 +73,6 @@ packages:
     externals:
     - spec: bzip2@1.0.6
       prefix: /usr
-  cmake:
```

Comment:
Can you remind me why this needs to be removed?

Reply:
I think this ended up being an issue for hdf5, related to the compiler settings. I haven't tested reincorporating it after we made the hdf5 package.py update, though I would be inclined to leave out external cmake on orion so that we have more consistent cmake version across platforms as much as possible.

Reply:
Ok, that's fine with me.